.github/CODEOWNERS

```diff
@@ -2,6 +2,5926 @@
 # Add yourself to the "Definitions by:" list instead.
 # See https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package
 
+<<<<<<< HEAD
```

Comment:
This file is updated automatically by a tool - please don't edit it directly.

Reply:
I think this is fixed now, thanks

Reply:
Also, yes, the intention is to deprecate the akumina-core NPM package and just release the types package for now...